tests/idem/structs.rs

```diff
@@ -14,6 +14,18 @@ pub struct Foo {
 
 struct Bar;
 
+struct NewInt<T: Copy>(i32, SomeType /* inline comment */, T);
+
+struct Qux<'a,
+           N: Clone + 'a,
+           E: Clone + 'a,
+           G: Labeller<'a, N, E> + GraphWalk<'a, N, E>,
+           W: Write + Copy>
+    (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, // Comment
+     BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB,
+     G,
+     W);
```

Comment:
That looks good, but isn't it inconsistent with the rest of the formatting rules? We usually don't have a 'dangling' closing paren for tuple-like structures.